server/dex/dex.go

```diff
@@ -298,12 +357,21 @@ func NewDEX(cfg *DexConf) (*DEX, error) {
 	now := encode.UnixMilli(time.Now())
 	bookSources := make(map[string]market.BookSource, len(cfg.Markets))
 	marketTunnels := make(map[string]market.MarketTunnel, len(cfg.Markets))
+	cfgMarkets := make([]msgjson.Market, 0, len(cfg.Markets))
 	for name, mkt := range markets {
 		startEpochIdx := 1 + now/int64(mkt.EpochDuration())
 		mkt.SetStartEpochIdx(startEpochIdx)
 		startSubSys(fmt.Sprintf("Market[%s]", name), mkt)
 		bookSources[name] = mkt
 		marketTunnels[name] = mkt
+		cfgMarkets = append(cfgMarkets, msgjson.Market{
+			Name:            name,
+			Base:            mkt.Base(),
+			Quote:           mkt.Quote(),
+			EpochLen:        mkt.EpochDuration(),
+			StartEpoch:      uint64(startEpochIdx),
+			MarketBuyBuffer: mkt.MarketBuyBuffer(),
+		})
```

Comment:
Stepping on your toes with #139 here.

Reply:
No problem.  I hadn't considered that this would hold you up.  My bad.